public/app/features/explore/TraceView/createSpanLink.tsx

```diff
@@ -383,10 +383,10 @@ function getQueryForLoki(
 
   let expr = '{${__tags}}';
   if (filterByTraceID && span.traceID) {
-    expr += ' |="${__span.traceId}"';
+    expr += ' | trace_id="${__span.traceId}"';
   }
   if (filterBySpanID && span.spanID) {
-    expr += ' |="${__span.spanId}"';
+    expr += ' | span_id="${__span.spanId}"';
```

Comment:
Ditto, doesn't this new syntax only work when logs are persisted with the Loki format for Otel logs relying on Loki log record metadata and labels?